### DISCUSSION TITLE
What does "classic" mean?

### DISCUSSION THREAD

**Discussion by TurnOffNOD** [Stellarium/stellarium#120] [Q&A] [answered]
Among the release installer files, there are two win32 installer: stellarium-xxx-win32.exe and stellarium-xxx-classic-win32.exe, what is the difference? Does "classic" mean it is for WinXP?

**Comment by alex-w** [comment, marked as answer]
Yes, "classic" edition is for Windows XP

**Comment by TurnOffNOD** [comment]
@alex-w Thanks for reply. There is no description in Wiki or FAQ, so I asked here. It should be made clear in FAQ and Wiki.
